src/__tests__/linter.test.ts

```diff
@@ -1202,13 +1202,13 @@ responses:: !!foo
       ).resolves.toEqual([
         expect.objectContaining({
           code: 'header-parameter-names-kebab-case',
-          message: '"fooA" is not kebab-cased: must match the pattern \'^[a-z0-9]+((-[a-z0-9]+)+)?$\'',
+          message: '"fooA" is not kebab-cased: "fooA" must match the pattern "^[a-z0-9]+((-[a-z0-9]+)+)?$"',
           path: ['parameters', '0', 'name'],
         }),
 
         expect.objectContaining({
           code: 'header-parameter-names-kebab-case',
-          message: '"d 1" is not kebab-cased: must match the pattern \'^[a-z0-9]+((-[a-z0-9]+)+)?$\'',
+          message: '"d 1" is not kebab-cased: "d 1" must match the pattern "^[a-z0-9]+((-[a-z0-9]+)+)?$"',
```

Comment:
Do we ned to have `d 1` in the message twice?

Reply:
It's just for the sake of testing.
The test verifies we evaluate `{{<pattern>}}`, thus our rule has a custom message.

```ts
describe('evaluate {{value}} in validation messages', () => {
  test('should print primitive values', () => {
    spectral = new Spectral();
    spectral.setRules({
      'header-parameter-names-kebab-case': {
        severity: DiagnosticSeverity.Error,
        recommended: true,
        description: 'A parameter in the header should be written in kebab-case',
        message: '#{{print("value")}} is not kebab-cased: {{error}}', // this is why it's twice
```